app/models/expense.rb

```diff
@@ -7,4 +7,22 @@ class Expense < ActiveRecord::Base
   validates :description, :user, :amount, :paid_on, :category, presence: true
   validates :paid_on, date: { before_or_equal_to: proc { Date.today } }
   validates_associated :receipt
+
+  def format_amount
+    '€' + amount.to_s
+  end
+
+  def format_vat
+    vat.to_s + '%' unless vat.nil?
+  end
+
+  def format_amount_w_vat
+    '€' + amount_w_vat.to_s unless vat.nil?
+  end
```

Comment:
Yes, you're right. We could have another object with this behavior to sit between the model and the view (presenter, viewmodel, ...).
Nevertheless, this fits the purpose for now so let's leave it. :+1: